packages/pipeline/samples/java/project.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "pipeline-sample-java",
+  "root": "packages/pipeline/samples/java",
+  "implicitDependencies": [
+    "aws-prototyping-sdk"
+  ],
+  "targets": {
+    "clobber": {
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "pnpx  projen clobber",
```

Comment:
Nitpick: extra space after "pnpx" in following commands: `pnpx  projen clobber`

Reply:
Don't worry about this one - I have fixed in my PR

Reply:
Sorry - already fixed :)